### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by glaucocustodio**
I think nice to show we can reuse formatters.

### CODE REVIEW

**Comment by dblock** [comment]
Thanks. Maybe combine it with the example above and add some explanations in plain English?

**Comment by glaucocustodio** [comment]
How about that @dblock:

Apply a formatter before exposing a value.

```ruby
format_with(:iso_timestamp) { |dt| dt.iso8601 }
with_options(format_with: :iso_timestamp) do
  expose :created_at
  expose :updated_at
end
```

Defining a reusable formatter between multiples entities:

```ruby
# Helper module for instance
Grape::Entity.format_with :utc do |date|
  date.utc if date
end
 
# Entity
expose :updated_at, format_with: :utc
```

**Comment by dblock** [comment]
The language is fine, but I'd like the two code examples to be consistent, for example the first doesn't have `Grape::Entity.` and the second does. One uses a block with `do` and the other one has `{`.

**Comment by glaucocustodio** [comment]
Just `format_with` inside a helper raises an exception like `undefined method format_with for ApiHelpers:Module`, that is the reason for `Grape::Entity.` before. Maybe we can give a bit more of context like below. `{` is the proper style for inline blocks, we can change to a multi line block if necessary (I don't think so).

Apply a formatter before exposing a value.

```ruby
module Entities
  class MyModel < Grape::Entity
    format_with(:iso_timestamp) { |dt| dt.iso8601 }

    with_options(format_with: :iso_timestamp) do
      expose :created_at
      expose :updated_at
    end    
  end
end
```

Defining a reusable formatter between multiples entities:

```ruby
module ApiHelpers
  extend Grape::API::Helpers

  Grape::Entity.format_with :utc do |date|
    date.utc if date
  end
end
```

```ruby
module Entities
  class MyModel < Grape::Entity
    expose :updated_at, format_with: :utc
  end

  class AnotherModel < Grape::Entity
    expose :created_at, format_with: :utc
  end
end
```

**Comment by dblock** [comment]
Much better, amend and I'll merge.

**Comment by dblock** [comment]
I still have major ADD around `{` vs. `do` btw!

**Comment by glaucocustodio** [comment]
Please, check now.

**Comment by dblock** [comment]
Looks great, merging.
